include/zephyr/bluetooth/bluetooth.h

```diff
@@ -48,6 +49,13 @@ extern "C" {
  */
 #define BT_ID_DEFAULT 0
 
+/**
+ * @brief Number of octets for local supporte
+ *
+ * The value of 8 correspond to page 0 in the LE Controller supported features
+ */
+#define BT_LE_LOCAL_SUPPOTRED_FEATURES_SIZE 8
```

Comment:
Do we need this define in the first place or can we hardcode it in the definition of `struct bt_le_local_features`. If we hardcode it there, we reduce the amount of documentation needed.

We could consider adding ifdefs in the struct definition to remove entries which are not needed.

Reply:
Yeah I didn't write that in the commit message. 

The reason why I've added it here, is that today the Zephyr only supports page 0, but there are actually more than 1 page now. 

Once we add support for reading all pages, then it would be trivial to simply increase this from e.g. 8 to 16.

If that doesn't really make sense, let me know :)

Reply:
Hmm, perhaps it should be an internal Kconfig option instead?

Reply:
I was wondering if we need an entry at all: `uint8_t features[8];`. A Kconfig option is probably not needed

Reply:
I just really dislike magic numbers in the public API like `8` would be :D (and especially when it should fit the same size of a different struct's field)

Could add a define for PAGE_SIZE in `hci_types.h`, but that doesn't solve anything either.

Reply:
I see that it is also used inside hci_core.h, so it is probably a good idea to keep it as is